src/backend/core/api/viewsets/config.py

```diff
@@ -137,6 +141,7 @@ def get(self, request):
                         "sdk_url": f"{base_url}{settings.DRIVE_CONFIG.get('sdk_url')}",
                         "api_url": f"{base_url}{settings.DRIVE_CONFIG.get('api_url')}",
                         "file_url": f"{base_url}{settings.DRIVE_CONFIG.get('file_url')}",
+                        "app_name": settings.DRIVE_APP_NAME,
```

Comment:
According to my comment into `settings.py`


```suggestion
                        "app_name": settings.DRIVE_CONFIG.get('app_name'),
```